module/icp/algs/skein/THIRDPARTYLICENSE

```diff
@@ -0,0 +1,3 @@
+Implementation of the Skein hash function.
+Source code author: Doug Whiting, 2008.
+This algorithm and source code is released to the public domain.
```

Comment:
Technically, the author should use a real license grant. It's (probably; there is a minority of experts who disagree) not possible to voluntarily place code in the public domain in the US. In practice, it's probably fine.